airbyte-integrations/connectors/source-jira/integration_tests/invalid_config.json

```diff
@@ -1,6 +1,5 @@
 {
   "api_token": "invalid_token",
-  "domain": "invaliddomain.atlassian.net",
```

Comment:
why did you remove domain from here?

Reply:
because this domain do not exist, and for this case we need to raise a config error, which not return (False, reason) as CAT expects, so to pass this test I removed one item to make this config invalid.